content/docs/tour/basics-projects.md

```diff
@@ -1,100 +1,3 @@
 ---
-title: Creating a project
-menu:
-  tour:
-    parent: tour
-    weight: 2
+redirect_to: https://www.pulumi.com/docs/reference/project/
```

Comment:
Can this be a relative URL instead?

Reply:
Yes. This should be a relative link. I missed cleaning this up. Will fix.